lib/xwax/0005-Fix-traktor-math-issues-for-timecoder-bad-values-lp-.patch

```diff
@@ -0,0 +1,29 @@
+From 945947d856710ba7aeadc985734ef991efdaa437 Mon Sep 17 00:00:00 2001
+From: Owen Williams <owilliams@mixxx.org>
+Date: Sun, 2 Jan 2022 02:16:51 +0100
+Subject: [PATCH 5/5] Fix traktor math issues for timecoder bad values
+ (lp:788630)
+
+---
+ timecoder.c | 5 +++++
+ 1 file changed, 5 insertions(+)
+
+diff --git a/timecoder.c b/timecoder.c
+index 63a9a27..da89655 100755
+--- a/timecoder.c
++++ b/timecoder.c
+@@ -640,6 +640,11 @@ signed int timecoder_get_position(struct timecoder *tc, double *when)
+     if (r == -1)
+         return -1;
+ 
++    if (r >= 0) {
```

Comment:
Mark has told me that we are using his library wrong, and this is probably evidence of that.  This isn't something to fix now, but we should look at it in the future.

Reply:
maybe leave a TODO comment?